tests/scripts/nvidia-kernel-upgrade-aws.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+if [[ "${SKIP_INSTALL}" == "true" ]]; then
+    echo "Skipping install: SKIP_INSTALL=${SKIP_INSTALL}"
+    exit 0
+fi
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "${SCRIPT_DIR}"/.definitions.sh
+
+# finding kernel version
+${SCRIPT_DIR}/findkernelversion.sh
+source "${LOG_DIR}"/kernel_version.txt
+
+echo "Checking current kernel version..."
+CURRENT_KERNEL=$(uname -r)
+echo "Current kernel version: $CURRENT_KERNEL"
+
+if [ "${CURRENT_KERNEL}" != ${KERNEL_VERSION} ]; then
+  echo ""
+  echo ""
+  echo "--------------Starting the Precompiled kernel version ${KERNEL_VERSION} upgrade--------------"
+
+  sudo apt-get update -y
+  sudo apt-get install --allow-downgrades linux-image-${KERNEL_VERSION} -y
+  if [ $? -ne 0 ]; then
+    echo "Kernel upgrade failed."
+    exit 1
+  fi
+
+  echo "update grub ..."
+  sudo sed -i "s/^GRUB_DEFAULT=.*/GRUB_DEFAULT=\"Advanced options for Ubuntu>Ubuntu, with Linux ${KERNEL_VERSION}\"/" /etc/default/grub
```

Comment:
Is there an identifier we can use instead of this string? `Advanced options for Ubuntu>Ubuntu, with Linux ${KERNEL_VERSION}`

Reply:
added logic to calculate submenu ->menuentry of upgrade kernel . 
GRUB_DEFAULT="1>4"     (for kernel version upgrade